docs/tutorial/getting-started.md

```diff
@@ -1,3 +1,210 @@
+
+If you want, you can build and install Pebble from source:
+
+1. Follow the official Go documentation [here](https://go.dev/doc/install) to download and install Go.
+2. After installing, you will want to add the `$GOBIN` directory to your `$PATH` so you can use installed tools. For more information, refer to the [official documentation here](https://go.dev/doc/install/source#environment).
+3. Run `go install github.com/canonical/pebble/cmd/pebble@latest` to build and install Pebble.
+
+### 2.3 Sanity check
+
+After installation, if you run `pebble`, you should see some output equivalent to the following:
+
+```bash
+$ pebble
+Pebble lets you control services and perform management actions on
+the system that is running them.
+
+Usage: pebble <command> [<options>...]
+
+Commands can be classified as follows:
+
+       Run: run, help, version
+      Plan: add, plan
+  Services: services, logs, start, restart, signal, stop, replan
+    Checks: checks, health
+     Files: push, pull, ls, mkdir, rm, exec
+   Changes: changes, tasks
+   Notices: warnings, okay, notices, notice, notify
+
+Set the PEBBLE environment variable to override the configuration directory
+(which defaults to /var/lib/pebble/default). Set PEBBLE_SOCKET to override
+the unix socket used for the API (defaults to $PEBBLE/.pebble.socket).
+
+For more information about a command, run 'pebble help <command>'.
+For a short summary of all commands, run 'pebble help --all'.
+```
+
+Pebble is invoked using `pebble <command>`. To get more information:
+
+- To see a help summary, type `pebble -h`.
+- To see a short description of all commands, type `pebble help --all`.
+- To see details for one command, type `pebble help <command>` or `pebble <command> -h`.
+
+## 3 Configure Pebble
+
+First, create a directory for Pebble configuration:
+
+```bash
+$ mkdir -p ~/PEBBLE/layers
+$ export PEBBLE=$HOME/PEBBLE
+$ echo "export PEBBLE=$HOME/PEBBLE" >> ~/.bashrc # add $PEBBLE to your bashrc
+```
+
+Then, create a simple layer:
+
+```bash
+$ echo """\
+summary: Simple layer
+
+description: |
+    A simple layer.
+
+services:
+    http-server:
+        override: replace
+        summary: demo http server
+        command: python3 -m http.server 8080
+        startup: enabled
+""" > $PEBBLE/layers/001-http-server.yaml
+```
+
+This creates a simple layer containing only one service (which runs a basic HTTP server using Python's `http` module, listening on port 8080).
+
+## 4 Run the Pebble daemon
+
+```bash
+$ pebble run
+2024-06-02T11:30:02.925Z [pebble] Started daemon.
+2024-06-02T11:30:02.936Z [pebble] POST /v1/services 10.751704ms 202
+2024-06-02T11:30:02.936Z [pebble] Started default services with change 77.
+2024-06-02T11:30:02.942Z [pebble] Service "http-server" starting: python3 -m http.server 8080
+```
+
+This starts the Pebble daemon, and as you can see from the log, our HTTP server is already started, which can be verified by running `curl localhost:8080` in another terminal tab.
```

Comment:
Should we add a paragraph (sentence) after this saying "To exit the Pebble daemon, press Ctrl-C (which sends an "interrupt" signal to the process)." or similar.